plugins/search-react/package.json

```diff
@@ -43,6 +42,7 @@
     "react": "^16.13.1 || ^17.0.0"
   },
   "devDependencies": {
+    "@backstage/core-app-api": "^1.0.1",
```

Comment:
can't storybook use the TestApiProvider? 🤔  isn't that the only active use

Reply:
Either or really, not too big of a deal as long as `core-app-api` is a dev dep